backend/src/org/akvo/dash/endpoint/public.clj

```diff
@@ -30,10 +30,11 @@
 
 (defn- visualisation-id-list
   [dashboard]
-  (remove nil? (mapv (fn [m]
-                       (if (= (get m "type") "visualisation")
-                         (get m "id")))
-                     (vals (:entities dashboard)))))
+  (remove nil? (->> dashboard
```

Comment:
There will be no `nil` values since you're filtering by `"visualisation"` ? so no need to `(remove nil? coll)` ?